javascript/nodejs/express_bodyparser_dos.yaml

```diff
@@ -0,0 +1,12 @@
+rules:
+  - id: express_bodyparser
+    pattern: $X.bodyParser(...)
```

Comment:
Should this be inside a `pattern-inside`?

Reply:
Yes